web-api/src/error/common.rs

```diff
@@ -68,10 +72,27 @@ pub(crate) struct InvalidDocumentPropertyId {
 
 impl_application_error!(InvalidDocumentPropertyId => BAD_REQUEST, INFO);
 
-/// Malformed document property.
+/// Malformed property {document}/{property}, {invalid_reason}: {invalid_value}
 #[derive(Debug, Error, Display, Serialize)]
+// there are some false positives with clippy and displaydoc
+#[allow(clippy::doc_markdown)]
 pub(crate) struct InvalidDocumentProperty {
-    pub(crate) value: Value,
+    pub(crate) document: DocumentId,
+    pub(crate) property: DocumentPropertyId,
+    pub(crate) invalid_value: Value,
+    pub(crate) invalid_reason: InvalidDocumentPropertyReason,
```

Comment:
:art: we could declare them in the order of usage
```suggestion
    pub(crate) invalid_reason: InvalidDocumentPropertyReason,
    pub(crate) invalid_value: Value,
```

Reply:
I'm referring the all the places in the code we create the struct :sweat_smile: